src/hooks/useDrivesContainer.ts

```diff
@@ -2,12 +2,17 @@ import {
     BaseTreeItem,
     DriveViewProps,
     TreeItem,
+    SharingType as TreeItemSharingType,
```

Comment:
would you prefer us to rename this type to be more specific like this in the design system code?

Reply:
I think the best fix is to change the sharing type in the document model to also be uppercase.